hipaa-password-input.html

```diff
@@ -3,8 +3,8 @@
 <link rel="import" href="../paper-input/paper-input-behavior.html">
 <link rel="import" href="../paper-icon-button/paper-icon-button.html">
 <link rel="import" href="../iron-icon/iron-icon.html">
-<link rel="import" href="../iron-icons/iron-icons.html">
 <link rel="import" href="../paper-input/paper-input-error.html">
+<link rel="import" href="common/hipaa-password-icons.html" async>
```

Comment:
pls change this to:
`<link rel="import" href="hipaa-password-icons.html" async>`